examples/start_example_server.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+import os
+import sys
+import logging
+import webbrowser
+
+PORT=8080
+STATIC_RESOURCES = { "/wasm.js", "/wasm_bg.wasm", "/favicon.ico" }
+DEFAULT_INDEX_HTML = """<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Yew example</title>
+        <script type="module">
+            import init from "./wasm.js"
+            init()
+        </script>
+    </head>
+    <body></body>
+</html>""".encode('utf-8')
+
+class S(BaseHTTPRequestHandler):
+    def _set_response(self, code, mime, charset='utf-8'):
+        self.send_response(code)
+        self.send_header('Content-type', f'{mime}; charset={charset}')
+        self.send_header('Cache-Control', 'private, max-age=0, must-revalidate')
+        self.end_headers()
+
+    def do_GET(self):
+        path = self.path
+        if path in ('', '/'):
+            path = '/index.html'
+        relative_path = f'.{path}'
+
+        if os.path.isfile(relative_path):
+            print('is a file:', relative_path)
```

Comment:
We already have the `logging` module imported.
I think the debug level would be more appropriate for this but since the log level isn't configurable I went with info:
```suggestion
            logging.info("is a file: %s", relative_path)
```